.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
And @Mixih without grep it will be like this
![image](https://cloud.githubusercontent.com/assets/17270966/22261543/7086b5cc-e293-11e6-969f-bdacd6dc0505.png)